freqtrade/freqtradebot.py

```diff
@@ -480,6 +480,14 @@ def execute_buy(self, pair: str, stake_amount: float, price: Optional[float] = N
         else:
             # Calculate price
             buy_limit_requested = self.exchange.get_rate(pair, refresh=True, side="buy")
+            if self.config.get('use_custom_entry_price', False):
```

Comment:
I'm quite hesitant about `use_custom_entry_price`.

The default will not change behaviour (so if a strategy doesn't implement this - it'll simply return poposed_rate) - so it might be "always on" - and the opt-in is by overwriting the method in your strategy.

Having another setting is not bad per se - but i feel it's unnecessary in this instance.

Reply:
Good suggestion